### ISSUE TITLE
Convert integration tests to use ISLE, not the Playbook

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#487]
Since ISLE appears to be the preferred method of running an Islandora instance locally, tests should use it, and not the Playbook, as their target Drupal.

Currently, all integration tests use the following values in their config files:

```yaml
host: "http://localhost:8000"
username: admin
password: islandora
```

They should use:

```yaml
host: https://islandora.traefik.me/
username: admin
password: password
```
These credentials assume that the ISLE "secrets" are not enabled.

**Comment by mjordan** [comment]
Running `--check` errors out with "Error: Field "field_resource_type" required for content type "islandora_object" is not present in the CSV file." Every config file will need to have this added using a field template.

**Comment by mjordan** [comment]
Turns out that localhost:8000 and password "islandora" work with ISLE.... so no need to change those settings.

**Comment by mjordan** [comment]
Specifically, the ISLE Starter Site.

**Comment by mjordan** [comment]
Done so far:

- TestCreate
- TestCreateFromFiles
- TestDelete
- TestUpdate
- TestCreateWithNonLatinText

**Comment by mjordan** [comment]
tests/islandora_tests.py complete

**Comment by ysuarez** [comment]
@mjordan now that I have been slowly reading and learning the workbench codebase, I may be able to help a bit with this issue. Let me know here or on slack how I can start helping you with this issue.

**Comment by mjordan** [comment]
Resolved with 47aa7ea164063716bfbef5c04eb7ff0e46694dd1.
